src/client/providers/setInterpreterProvider.ts

```diff
@@ -129,6 +129,6 @@ function setInterpreter() {
             () => {
                 // The user doesn't have any workspace settings!
                 // Prompt them to create one first
-                vscode.window.showInformationMessage("No workspace settings file. First, run 'Preferences: Open Workspace Settings' to create one." )
+                vscode.window.showErrorMessage("No workspace settings file. First, run 'Preferences: Open Workspace Settings' to create one." )
```

Comment:
You can add this later (no rush):  
- I believe invoking the command 'workbench.action.openWorkspaceSettings' will create the necessary file.  

Once again this is not necessary for now, (nice to have, but not important).